github-issues.el

```diff
@@ -225,8 +225,8 @@
     (message "No current issue selected")))
 
 (defvar github-issues-mode-map
-  (let ((map (make-keymap)))
-    (define-key map "\C-cr" 'github-issues-refresh)
+  (let ((map (make-sparse-keymap)))
+    (define-key map "g" 'github-issues-refresh)
```

Comment:
What is the difference between `make-keymap` and `make-sparse-keymap`? `M-x describe-function` wasn't very useful for me.